pkg/doc/signature/verifier/verifier.go

```diff
@@ -10,15 +10,33 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/jsonld"
+
 	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/proof"
 )
 
+// nolint:gochecknoglobals
+var (
+	// CredentialStatusList2017 from credential status type is treated as invalid RDF
+	//  and excluded in signature verification, but it needs to be included while verifying signature.
+	// TODO to be removed as part of [Issue#1697]
+	allowedInvalidRDF = []string{
```

Comment:
seems like this might be an option that is passed-in. alternatively, maybe we could pass in additional context parameter argument to make the RDF valid... without needing to host it.